do cheesewheel items

Cheesewheel8686 opened the pull request:
Added PFE R-40-23 Ice Reefer & Textures

Cheesewheel8686 committed:
Add Refrigerated Car to JCIR

-Added PFE R-40-23 Ice Reefer
-Added Skins for PFE R-40-23

Cheesewheel8686 committed:
Merge branch 'foxtc-020-tracktastic' of https://github.com/FloofyPeachy/Traincraft-5 into foxtc-020-tracktastic

RealTBEA commented on src/main/java/com/jcirmodelsquad/tcjcir/models/trains/ModelPFEr_40_23Reefer.java:
```diff
@@ -0,0 +1,580 @@
+	@Override
+	public void render(Entity entity, float f, float f1, float f2, float f3, float f4, float f5)
+	{
+		for(int i = 0; i < 122; i++)
```
ModelRenderHelper.renderModelWithStandardFreightRollingStock(bodyModel, entity, f5);

Cheesewheel8686 replied:
complete

Cheesewheel8686 committed:
modify Ma&Pa Truck Position

modified Ma&pa flat & gon truck positions

Cheesewheel8686 committed:
She Southern on my South till i Serve

-added southern texture to SD40-2
-added horn sound to Seaboard SD40-2
-added southern texture to PS2 4750 Hopper

Cheesewheel8686 commented on src/main/java/train/common/library/EnumSounds.java:
```diff
@@ -161,8 +161,8 @@ public enum EnumSounds {
-	DieselSD40dash2(com.jcirmodelsquad.tcjcir.vehicles.locomotives.DieselSD40dash2.class, " ", 1F, "EMD_16_645E3_Notch8", 0.45F, 51, "EMD_16_645E3_Idle", 0.45F, 40, false, "emd_steelbell_8", 13, new String[] {"leslie_s3l_3", "leslie_rs3k_1", "nathan_p5_old_1", "nathan_p5_new_2", "leslie_s3l_2", "nathan_k3la_2", "nathan_k3la_4", "leslie_rs3l_3", "leslie_rs3l_3", "leslie_rs3l_5", "nathan_p5_new_1", "l ...
-	DieselSD40T2(com.jcirmodelsquad.tcjcir.vehicles.locomotives.DieselSD40T2.class, " ", 1F, "EMD_16_645E3_Notch8", 0.45F, 51, "EMD_16_645E3_Idle", 0.45F, 40, false, "emd_steelbell_1", 15, new String[] {"nathan_m3h_3", "nathan_p3_old_4", "nathan_p3_old_1", "nathan_p3_new_2", "nathan_k3la_3", "generic_horn_2", "nathan_p3_old_5", "generic_horn_1", "buell_b5_1", "nathan_j3_2", "generic_horn_2", "leslie ...
+	DieselSD40dash2(com.jcirmodelsquad.tcjcir.vehicles.locomotives.DieselSD40dash2.class, " ", 1F, "EMD_16_645E3_Notch8", 0.45F, 51, "EMD_16_645E3_Idle", 0.45F, 40, false, "emd_steelbell_8", 13, new String[] {"leslie_s3l_3", "leslie_rs3k_1", "nathan_p5_old_1", "nathan_p5_new_2", "leslie_s3l_2", "nathan_k3la_2", "nathan_k3la_4", "leslie_rs3l_3", "leslie_rs3l_3", "leslie_rs3l_5", "nathan_p5_new_1", "l ...
+	DieselSD40T2(com.jcirmodelsquad.tcjcir.vehicles.locomotives.DieselSD40T2.class, " ", 1F, "EMD_16_645E3_Notch8", 0.45F, 51, "EMD_16_645E3_Idle", 0.45F, 40, false, "emd_steelbell_1", 15, new String[] {"nathan_m3h_3", "nathan_p3_old_4", "nathan_p3_old_1", "nathan_p3_new_2", "nathan_k3la_3", "generic_horn_2", "nathan_p3_old_5", "generic_horn_1", "buell_b5_1", "nathan_j3_2", "generic_horn_2", "leslie ...
```
unnecessary comma

Cheesewheel8686 committed:
Update EnumSounds.java

-removed unneeded comma

RealTBEA commented:
This is a newer thing but convert this to use AbstractStandardFreightCar see ACF3500 as a example.

Cheesewheel8686 commented:
> This is a newer thing but convert this to use AbstractStandardFreightCar see ACF3500 as a example.

for P40-23?

Cheesewheel8686 committed:
update some file

add texture descriptions for SD40-2 and PS2 4750

Cheesewheel8686 committed:
Update PS2_4750.java

fix skill issue

Cheesewheel8686 committed:
Merge branch 'FloofyPeachy:foxtc-020-tracktastic' into foxtc-020-tracktastic

Cheesewheel8686 committed:
fix stupid files

fix stupid files for stupid boxcar
